src/App.tsx

```diff
@@ -26,70 +27,107 @@ type ReorderOptions = {
-  return (
-    <div className="section content">
-      <div className="buttons">
-        <button
-          type="button"
-          className="button is-info is-light"
-        >
-          Sort alphabetically
-        </button>
-
-        <button
-          type="button"
-          className="button is-success is-light"
-        >
-          Sort by length
-        </button>
-
-        <button
-          type="button"
-          className="button is-warning is-light"
-        >
-          Reverse
-        </button>
-
-        <button
-          type="button"
-          className="button is-danger is-light"
-        >
-          Reset
-        </button>
-      </div>
+type State = {
+  isReversed: boolean,
+  sortType: SortType,
+};
+
+export class App extends React.Component {
+  state: State = {
+    isReversed: false,
+    sortType: SortType.NONE,
+  };
+
+  setSortType = (sortType: SortType) => {
+    this.setState({ sortType });
+  };
+
+  setReverse = () => {
+    this.setState((Prev: State) => ({ isReversed: !Prev.isReversed }));
+  };
+
+  render() {
+    const { sortType, isReversed }: ReorderOptions = this.state;
+
+    return (
+      <div className="section content">
+        <div className="buttons">
+          <button
+            type="button"
+            className={classNames('button',
+              'is-info',
+              { 'is-light': sortType !== SortType.ALPHABET })}
+            onClick={() => this.setSortType(SortType.ALPHABET)}
+          >
+            Sort alphabetically
+          </button>
+
+          <button
+            type="button"
+            className={classNames('button',
+              'is-info',
+              { 'is-light': sortType !== SortType.LENGTH })}
+            onClick={() => this.setSortType(SortType.LENGTH)}
+          >
+            Sort by length
+          </button>
+
+          <button
+            type="button"
+            className={classNames('button',
+              'is-info', { 'is-light': isReversed === false })}
+            onClick={() => this.setReverse()}
+          >
+            Reverse
+          </button>
+
+          {(sortType !== 0 || isReversed) && (
+            <button
+              type="button"
+              className="button is-danger is-light"
+              onClick={() => {
+                this.setState({ isReversed: false });
+                this.setSortType(SortType.NONE);
+              }}
+            >
+              Reset
+            </button>
+          )}
+        </div>
 
-      <ul>
         <ul>
-          <li data-cy="Good">Dumplings</li>
-          <li data-cy="Good">Carrot</li>
-          <li data-cy="Good">Eggs</li>
-          <li data-cy="Good">Ice cream</li>
-          <li data-cy="Good">Apple</li>
-          <li data-cy="Good">...</li>
+          <ul>
+            {getReorderedGoods(goodsFromServer,
+              { sortType, isReversed }).map(good => (
+            // eslint-disable-next-line @typescript-eslint/indent
```

Comment:
I think you can find a solution without disabling eslint here

Reply:
when I add spaces, it asks me to remove them and so on